src/components/Blacklist/index.tsx

```diff
@@ -353,24 +410,33 @@ const BlacklistedItem = ({ item, revalidateList }: BlacklistedItemProps) => {
                       numeric="auto"
                     />
                   ),
-                  user: (
-                    <Link href={`/users/${item.user.id}`}>
-                      <span className="group flex items-center truncate">
-                        <CachedImage
-                          type="avatar"
-                          src={item.user.avatar}
-                          alt=""
-                          className="avatar-sm ml-1.5"
-                          width={20}
-                          height={20}
-                          style={{ objectFit: 'cover' }}
-                        />
-                        <span className="ml-1 truncate text-sm font-semibold group-hover:text-white group-hover:underline">
-                          {item.user.displayName}
+                  user:
+                    item.user != null ? (
```

Comment:
Please use `!==`

Reply:
I use != here to check for both null and undefined, instead of only null. Things would similarly break if the user was undefined.

Reply:
I changed this one to `!== undefined`, typescript didn't see `!item.user` as a guard against undefined for some reason. item.user has type `User | undefined`, so the compiler should throw a sufficient fit if someone tries to put null there.

Imo, I find `!== undefined` uglier than `!= null`, but I don't mind using `!==` here if it's preferred, it's y'alls repo after all.

Reply:
I'll use `item.user` here then. I prefer `!= null` personally, but it's y'alls repo and they both work the same :)

Reply:
Done!